olp-cpp-sdk-dataservice-write/test/unit/TestVolatileLayerClient.cpp

```diff
@@ -108,10 +108,15 @@ class VolatileLayerClientTestBase : public ::testing::TestWithParam<bool> {
 
 class VolatileLayerClientOnlineTest : public VolatileLayerClientTestBase {
  protected:
+  static std::shared_ptr<olp::http::Network> s_network;
```

Comment:
```suggestion
  // Static network instance is necessary as it needs to outlive any created clients.
  // This is a known limitation as triggered send requests capture the network instance inside the callbacks.
  static std::shared_ptr<olp::http::Network> s_network;
  
```

Add newline before `SetUpTestSuite()`